.github/workflows/default.yml

```diff
@@ -48,7 +48,7 @@ jobs:
       - name: Set up JDK
         uses: actions/setup-java@v5.0.0
         with:
-          java-version: 17
+          java-version: 21
```

Comment:
Why is this update included in the CMS update commit? It doesn't seem to be related to the update.